nixos/modules/services/networking/quassel.nix

```diff
@@ -25,12 +25,12 @@ in
 
       package = mkOption {
         type = types.package;
-        default = pkgs.kde4.quasselDaemon;
-        defaultText = "pkgs.kde4.quasselDaemon";
+        default = pkgs.quasselDaemon_qt5;
```

Comment:
I might be missing something here but it doesn't looks like `quasselDaemon_qt5` exists in `all-packages.nix`. I think this should `quasselDaemon` (which is still qt5).